sale_stock_picking_blocking/security/sale_stock_picking_blocking_security.xml

```diff
@@ -4,8 +4,7 @@
 <odoo noupdate="1">
     <record id="group_sale_delivery_block" model="res.groups">
         <field name="name">Release Delivery Block in Sales Orders</field>
-        <field name="category_id" ref="base.module_category_hidden" />
```

Comment:
:information_source: unused in 19.0, cf. account.group_delivery_invoice_address:
- 18.0: https://github.com/odoo/odoo/blob/18.0/addons/account/security/account_security.xml#L49-L52 
- 19.0: https://github.com/odoo/odoo/blob/19.0/addons/account/security/account_security.xml#L46-L48